docs/connect-to-lms.md

```diff
@@ -0,0 +1,38 @@
+## Integrating with a Learning Management System
+
+GitHub Classroom is continuously adding support for integrating with your institution's Learning Management System (LMS)!
+Connecting GitHub Classroom to a Learning Management System will unlock new capabilities that will streamline and enhance
+the same workflow your used to on GitHub Classroom, such as:
+
+- Automatic roster provisioning
+- Assignment creation (Planned)
+- Submission syncing (Planned)
+
+### Supported Learning Management Systems
+
+GitHub Classroom integrates with your Learning Management via the industry-standard LTI protocol and is certified by the
+Instructional Management System Global Learning Consortium, so you can be confident your information is safe and secure. Any
+Learning Management System which is compliant with the LTI standard is compatible with GitHub Classroom, such as Blackboard,
```

Comment:
Blackboard is no longer supported. Can we add a notice somewhere that Blackboard is currently not supported?